modules/Microsoft.Network/dnsResolvers/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,60 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.network.dnsResolvers-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dnsrcom'
```

Comment:
```suggestion
param serviceShort string = 'ndrcom'
```

Reply:
NOTE: This required you to regenerate the ReadME

Reply:
Resolved in latest commit